src/Model/Resolver/Products/DataPostProcessor/Attributes.php

```diff
@@ -384,6 +385,9 @@ protected function isAttributeSkipped(
          * Don't skip if attribute is for the compare page
          */
         if (in_array($attribute->getId(), $this->configurableAttributeIds)) {
+            if(!$attribute->getUsedInProductListing() && !$isSingleProduct && !$isCartProduct){
+                    return true;
```

Comment:
Please put the correct spaces here.
1. The space between `){` is required.
2. There are too many spaces before the `return`.